### PR TITLE
WIP - EDU Course work

### PR DESCRIPTION
**Pull request by dorusomcutean**

### CODE REVIEW

File: resources/js/layouts/customer/CustomerLayout.vue [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            id="website-layout"
+        >
+            <slot />
+        </main>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: "WebsiteLayout",
```
Comment (LiamThursfield-NexusPoint) [inline resources/js/layouts/customer/CustomerLayout.vue]:
```suggestion
        name: "CustomerLayout",
```

**Review by LiamThursfield-NexusPoint — APPROVED** [review]

**Review by LiamThursfield — APPROVED** [review]

File: app/Http/Controllers/Admin/EDU/Program/ProgramController.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Http\Controllers\Admin\EDU\Program;
```
Comment (LiamThursfield-NexusPoint) [inline app/Http/Controllers/Admin/EDU/Program/ProgramController.php]: Is it program or programme?
I think british is programme in relation to e.g. a course/study. And program in terms of a computer program/software dev programming

Dunna matter either way I guess as long as we're consistent

File: app/Http/Controllers/Admin/EDU/Program/ProgramController.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+use Inertia\Response;
+
+class ProgramController extends AdminController
+{
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->addMetaTitleSection('EDU');
+        $this->addMetaTitleSection('Courses');
```
Comment (LiamThursfield-NexusPoint) [inline app/Http/Controllers/Admin/EDU/Program/ProgramController.php]: This still wp -- lots of course references

File: app/Http/Controllers/Admin/EDU/Program/ProgramController.php (hunk 2 above)
Comment (dorusomcutean) [reply app/Http/Controllers/Admin/EDU/Program/ProgramController.php]: Lol, great point 😆 . I was too tired and tried to get the bulk of it done. Will go back and change it to programme.

**Pull request merged**